t/16-utils-runcmd.t

```diff
@@ -296,4 +295,49 @@ subtest 'saving needle via Git' => sub {
     #       it is tested here explicitly.
 };
 
+subtest 'signal guard aborts when git is disabled and do_cleanup is "no"' => sub {
+    package My::FakeSignalGuard {
+        use Mojo::Base -base, -signatures;
+
+        has 'abort' => sub { 0 };
+
+        sub new($class, @args) { bless {abort => 0}, $class; }
```

Comment:
There's no need for an extra constructor if you use `Mojo::Base -base;`. The attribute can be written simpler, too:
```suggestion
        use Mojo::Base -base, -signatures;
        has abort => 0;
```